connection_pool/connection_pool.go

```diff
@@ -266,8 +268,20 @@ func (connPool *ConnectionPool) Call(functionName string, args interface{}, user
 	return conn.Call(functionName, args)
 }
 
+// Call16 calls registered Tarantool function.
+// It uses request code for Tarantool 1.6, so result is converted to array of arrays.
+// Deprecated since tarantool 1.7.2.
```

Comment:
Let's change in all places:
```suggestion
// Deprecated since Tarantool 1.7.2.
```
becase we use `Tarantool` in the code:
```bash
$ grep "Tarantool 1" -r . | wc -l
48
```

Reply:
Updated.